src/Components/Main/Main.css

```diff
@@ -0,0 +1,12 @@
+/* background-img {
+  background-repeat: cover;
+} */
+.App-header {
```

Comment:
Go ahead and pop this into a separate Header.css file and import into Header.js, same thing for Footer and Main.